tests/foreman/ui/test_host.py

```diff
@@ -1500,14 +1504,44 @@ def test_global_registration_with_gpg_repo_and_default_package():
         4. open the global registration form and update the gpg repo and key
         5. check host is registered successfully with installed same package
         6. check gpg repo is exist in registered host
-
-    :CaseAutomation: ManualOnly
     """
+    with session:
+        repo_name = 'foreman_register'
+        repo_url = GR_YUM_REPO
+        repo_gpg_url = GR_YUM_REPO_GPG_URL
+        cmd = session.host.get_register_command(
+            {
+                'general.capsule': module_proxy.name,
+                'general.operating_system': module_os.title,
+                'advanced.activation_keys': module_activation_key.name,
+                'general.insecure': True,
+                'advanced.force': True,
+                'advanced.install_packages': 'mlocate vim',
+                'advanced.repository': repo_url,
+                'advanced.repository_gpg_key_url': repo_gpg_url,
+            }
+        )
+
+    # register host, check repos enabled and packages are installed
+    with VMBroker(nick='rhel7', host_classes={'host': ContentHost}) as client:
```

Comment:
Since we are getting only one host, we can use the fixture `rhel7_contenthost` instead of context manger.
I see two more instances in this PR where this could be changed.

Reply:
Thanks, I will update.